README.md

```diff
@@ -201,14 +201,6 @@ For further information on these topics, consult the [module API][30] documentat
 
 For examples, refer to the [development server code][31].
 
-### Styling
```

Comment:
I think you should still leave some summary notes here in the README, _nudging_ people as you suggested! Add a direct link to `docs/crossword-styling.md#overview` to lead them on to the detail :)

Reply:
I split the difference - pointed to the crossword-styling.md file while aiming to avoid duplication as much as possible.